src/index.html

```diff
@@ -3,12 +3,60 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home" class="logo">
+        <img src="./images/logo.png" alt="Logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#" class="box is-active">Apple</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#" class="box">Samsung</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#" class="box">Смартфоны и телефоны</a>
+          </li>
+
+          <li class="nav_item">
+            <a 
+              href="#" 
+              class="box" 
+              data-qa="hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#" class="box">Гаджеты</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#" class="box">Планшеты</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#" class="box">ФОТО</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#" class="box">ВИДЕО</a>
+          </li>
```

Comment:
make text uppercased using css
```suggestion
          <li class="nav_item">
            <a href="#" class="box">Фото</a>
          </li>

          <li class="nav_item">
            <a href="#" class="box">Видео</a>
          </li>
```